### PR TITLE
Long Live The Keiretsu: Adds the Zeng-Hu seniority mask

### PR DESCRIPTION
**Pull request by feartheblackout**
image
image
image
image
image
As most who are aware of the veterancy plate know, it currently covers the _entire_ front of the face, making it seem impractical (if not physiologically incompatible) and the butt of many jokes. This PR adds a less invasive alternative, makes both a variant of the integrated medical HUD to make it more practical, and adds unique versions for Vaurca Workers/Warriors, Bulwarks, and Skrell.

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

**Comment by feartheblackout** [comment]
I'd be fine with adding it as an alternative if that's the desire of the maintainers, as long as I can figure out what to call it and how to distinguish it from the original.

**Comment by DreamySkrell** [comment]
I'd be fine with that personally.

I'd just give the other plate a different name, like a synonym or something.

**Comment by NonQueueingMatt** [comment]
making it an alternative is cool for zenghu

**Review by Arrow768 — APPROVED** [review]

File: code/modules/organs/subtypes/augment/augments/zengu_plate.dm [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 	name = "zeng-hu veterancy plate"
 	desc = "A clear sign of Zeng-Hu's best, this plate bearing the company's symbol is installed on those who prove themselves in the hyper-competitive environment."
 	icon_state = "zenghu_plate"
 	on_mob_icon = 'icons/mob/human_races/augments_external.dmi'
 	sprite_sheets = list(
-		BODYTYPE_VAURCA_BULWARK = 'icons/mob/species/bulwark/augments_external.dmi'
+		BODYTYPE_VAURCA = 'icons/mob/species/vaurca/augments_external.dmi'
+	)
+
+/obj/item/organ/internal/augment/eye_sensors/medical/zenghu_plate/mask
```
Comment (DreamySkrell) [inline code/modules/organs/subtypes/augment/augments/zengu_plate.dm:10]: I could be blind but I don't see the new seniority plate added to the loadout

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**